src/dotnet/Azure.ClientSdk.Analyzers/Azure.ClientSdk.Analyzers.Tests/AZC0018Tests.cs

```diff
@@ -193,29 +215,28 @@ await Verifier.CreateAnalyzer(code)
         }
 
         [Fact]
-        public async Task AZC0018ProducedForMethodsWithParameterModel()
+        public async Task AZC0018ProducedForMethodsWithOperationOfPageableModel()
```

Comment:
Would it be possible to preserve the existing test and add a new one for OperationOfPageable, instead of replacing the existing test?

Reply:
AZC0018ProducedForMethodsWithParameterModel still exists below. I think it is a diff issue.